### DISCUSSION TITLE
Improve find-bar addon

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7074] [Ideas]
The find-bar addon is incredibly useful for my students, as I support a style of programming with many minimal scripts (and, then, many messages). Unfortunately the addon is not very user-friendly as the script containing with the desired element always shows up under the find-bar menu, so you have to close the menu and then restart the search if you didn't find the script you were looking for. My suggestions are (one of the following ones):
- reduce the menu height to just one line when you have identified the relevant event/var, and are clicking the left/right arrows in the menu to get to the interested script 
- move at the right of the menu the script related to what you were looking for 

A final alternative (not my favorite one, as I don't like when users have to remember key combinations) would be highlighting the script found even when the user is using ctrl-left/right to move to the previous/next script.

**Comment by DNin01** [comment]
Related: #5881
